framework/lib/components/toolbox/toolbox.tsx

```diff
@@ -117,9 +118,9 @@ export const Toolbox = ({
           >
             { isResizable && <ResizeHandle { ...resizeProps } /> }
             <FocusScope autoFocus={ autoFocus }>
-              <Box width="full" height="full">
+              <VStack width="full" height="full">
```

Comment:
also why this change

Reply:
This is because we want the display flex and column on the container, in order to make the flex-grow prop on the children work